graphics.py

```diff
@@ -17,6 +17,7 @@
 import matplotlib.pyplot as plt
 import pygame
 from matplotlib.dates import HourLocator, DateFormatter
+import pprint
```

Comment:
what's this for?  I don't see any usages.

Reply:
It is a debugging remnant. It can be removed. Sorry.